pipelinesScripts/pluginUpload.sh

```diff
@@ -99,7 +88,7 @@ copyToLatestDir () {
   pluginPath="$JFROG_CLI_PLUGINS_RT_REGISTRY_REPO/$JFROG_CLI_PLUGIN_PLUGIN_NAME"
   echo "Copy version to latest dir: $pluginPath"
 
-  ./jfrog rt cp "$pluginPath/$JFROG_CLI_PLUGIN_VERSION/(*)" "$pluginPath/latest/{1}" --flat --url="$JFROG_CLI_PLUGINS_RT_REGISTRY_URL" --access-token=$JFROG_CLI_PLUGINS_RT_REGISTRY_TOKEN
+  res=$(eval "./jfrog rt cp \"$pluginPath/$JFROG_CLI_PLUGIN_VERSION/(*)\" \"$pluginPath/latest/{1}\" --flat")
```

Comment:
Is this copy still needed if the plugin are being distributed to releases and downloaded from there?

If it is:
The result of the command doesn't seems to be used, any reason to insert into this eval?